### PR TITLE
Apply interest collecting

### PR DESCRIPTION
**Pull request by Delfil**
Collects the interestingness to display it on the scroll bar.

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/Wrapper.java [hunk 1]
```diff
@@ -26,11 +26,11 @@
 	List<Integer> outgoingWeight = new ArrayList<>();
 	
 	int previousNodesCount = -1;
-	int interest = 0;
+	float interest = 0;
 	
 	/**
-	 * @return the maximum number of base pairs that can be passed when
-	 *         following the graph from the start of this node to its end.
+	 * @return the maximum number of base pairs that can be passed when following the graph from the start of this node
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/Wrapper.java]: *Auto-generated comment*

    Line is longer than 120 characters (found 123).

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/Wrapper.java [hunk 2]
```diff
@@ -116,21 +114,23 @@ public int compareTo(Wrapper other) {
...
 	}
 	
+	public void multiplyInterest(float interest) {
+		this.interest *= interest;
+	}
+	
 	/**
-	 * Computes the longest path for each {@link Wrapper} in the list and
-	 * returns the length of the longest path found.
+	 * Computes the longest path for each {@link Wrapper} in the list and returns the length of the longest path found.
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/Wrapper.java]: *Auto-generated comment*

    Line is longer than 120 characters (found 123).

**Comment by borismattijssen** [comment]
Please fix checkstyle errors. Then it's fine.

**Comment by borismattijssen** [comment]
SingleWrapper constructor needs JavaDoc

File: src/main/java/tudelft/ti2806/pl3/data/wrapper/operation/interest/ComputeInterest.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+	 * 
+	 * @param wrapper
+	 *            the wrapper
+	 */
+	public static void compute(Wrapper wrapper) {
+		new CalculateWrapPressureInterest().calculate(wrapper, null);
+		new CalculateMultiplyWithContainerInterest().calculate(wrapper, null);
+		new CalculateSizeInterest().calculate(wrapper, null);
+		new CalculateNInterest().calculate(wrapper, null);
+	}
```
Comment (kwendel) [inline src/main/java/tudelft/ti2806/pl3/data/wrapper/operation/interest/ComputeInterest.java]: Why not make al these Calculate classes private to the method? Creating new ones everytime is a bit useless

**Comment by borismattijssen** [comment]
Please create tests to verify it actually works

**Comment by TomBrouws** [comment]
Looks good, tests are nice.

**Pull request merged**